base/applications/osk/lang/de-DE.rc

```diff
@@ -238,69 +238,69 @@ BEGIN
     PUSHBUTTON      ".",SCAN_CODE_104,404,98,15,15
     PUSHBUTTON      "-",SCAN_CODE_105,424,22,15,15
     PUSHBUTTON      "+",SCAN_CODE_106,424,41,15,35
-    PUSHBUTTON      "ent",SCAN_CODE_108,424,78,15,35
+    PUSHBUTTON      "Enter",SCAN_CODE_108,424,78,15,35
     CTEXT           "Num",IDC_STATIC,364,3,21,8
     CONTROL         "",IDC_LED_NUM,"Static",SS_CENTER|SS_NOTIFY,372,15,4,3
     CTEXT           "Caps",IDC_STATIC,389,3,21,8
     CONTROL         "",IDC_LED_CAPS,"Static",SS_CENTER|SS_NOTIFY,398,15,4,3
-    CTEXT           "Scroll",IDC_STATIC,414,3,21,8
+    CTEXT           "Rollen",IDC_STATIC,414,3,21,8
     CONTROL         "",IDC_LED_SCROLL,"Static",SS_CENTER|SS_NOTIFY,424,15,4,3
 END
 
 IDR_OSK_MENU MENU DISCARDABLE
 BEGIN
-    POPUP "File"
+    POPUP "Datei"
     BEGIN
-        MENUITEM "&Exit", IDM_EXIT
+        MENUITEM "&Beenden", IDM_EXIT
```

Comment:
Put the Ampersand after the B?

Reply:
I agree with you. It would probably make sense to keep the shortcuts as close as possible to the "original" english translation. We then should also consider to unify this in all application with a german translation. In "taskmgr" and "dxdiag" the translation also has &Beenden instead of B&eenden . Maybe this should be updated too.